flink-core/src/test/java/org/apache/flink/api/common/state/StateDescriptorTest.java

```diff
@@ -251,6 +253,15 @@ public void go() {
         threads.clear();
     }
 
+    @Test
+    public void testStateTTlConfig() {
+        ValueStateDescriptor<Integer> stateDescriptor =
+                new ValueStateDescriptor<>("test-state", IntSerializer.INSTANCE);
+        Assert.assertFalse(stateDescriptor.getTtlConfig().isEnabled());
+        stateDescriptor.enableTimeToLive(StateTtlConfig.DISABLED);
+        Assert.assertFalse(stateDescriptor.getTtlConfig().isEnabled());
```

Comment:
This does not seem to test anything.

If you remove any of the above three lines, the test can still pass?

Reply:
If without the modification in `StateDescriptor.java`, the line 261 would fail.
I think we could improve the testing to verify it could hanlde non-disabled state ttl config and then also handle disablaed ttl config.